xbmc/settings/lib/SettingSection.h

```diff
@@ -122,6 +122,7 @@ class CSettingCategory : public ISetting
   bool CanAccess() const;
 
   void AddGroup(const SettingGroupPtr& group);
+  void AddGroupToBegin(const SettingGroupPtr& group);
```

Comment:
`AddGroupToFront()`

Reply:
Yeah the STL has it all: `begin()` and `end()`, `front()` and `back()` but the most matching name here would be `push_front()` like in `std::list` and other containers. That's why I would to with `AddGroupToFront()`. It also reads more naturally IMO.